tsl/test/expected/cagg_joins-12.out

```diff
@@ -188,8 +202,8 @@ NOTICE:  refreshing continuous aggregate "cagg_on_cagg"
 HINT:  Use WITH NO DATA if you do not want to refresh the continuous aggregate on creation.
 DROP MATERIALIZED VIEW cagg_on_cagg CASCADE;
 NOTICE:  drop cascades to 2 other objects
-DETAIL:  drop cascades to table _timescaledb_internal._hyper_6_35_chunk
-drop cascades to table _timescaledb_internal._hyper_6_36_chunk
+DETAIL:  drop cascades to table _timescaledb_internal._hyper_7_37_chunk
+drop cascades to table _timescaledb_internal._hyper_7_38_chunk
```

Comment:
Maybe you want to turn off the detailed output for this part of the test. It will change more times, and potentially generate false positives. It is sufficient to know the number of objects dropped.